### ISSUE TITLE
ImportError: cannot import name '_fftpack'

### ISSUE THREAD

**Issue by mariokreutzfeldt** [labsyspharm/ashlar#59]
Hi,

I installed ashlar in a conda environment. When I run ashlar -h I get the following error:

```
from . import scipy_fftpack
  File "...lib/python3.6/site-packages/pyfftw/interfaces/scipy_fftpack.py", line 65, in <module>
    from scipy.fftpack import (dct, idct, dst, idst, diff, tilbert, itilbert,
ImportError: cannot import name '_fftpack'
```
Scipy is installed. Running `from scipy.fftpack import fft` raises no error.

Helpful for any suggestions.

Thanks,
Mario

**Comment by jmuhlich** [comment]
Downgrading scipy to 1.3 should fix that -- As of Ashlar v1.9 we now explicitly specify this in setup.py. 

The problem is that pyfftw touches a semi-internal scipy interface, but that interface changed in scipy 1.4. To demonstrate the problem you would actually run `from scipy.fftpack import _fftpack`.

**Comment by mariokreutzfeldt** [comment]
@jmuhlich Thank you!
